ajax/libs/art-template/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "art-template",
+  "version": "4.9.1",
+  "filename": "template-web.js",
+  "description": "JavaScript template engine",
+  "homepage": "https://aui.github.io/art-template/",
+  "keywords": [
+    "template",
+    "engine",
+    "art"
```

Comment:
I think if add the followings keywords can let it better.
```js
    "util",
    "functional",
```
ref: https://github.com/aui/art-template/blob/3.0.1/package.json#L6-L7